<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addMatch</class>
 <widget class="QDialog" name="addMatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>160</width>
     <height>165</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Team 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTeam1"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spin1Score"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Team 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTeam2"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spin2Score"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Winner:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboWinner"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Best of:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBO">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="butComplete">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
